Gemfile.lock

```diff
@@ -30,17 +30,18 @@ PATH
 GEM
   remote: https://rubygems.org/
   specs:
-    activesupport (6.1.7.6)
```

Comment:
This branch shouldn't be changing `Gemfile.lock`, `package.json`, or `package-lock.json`.

Reply:
How should I resolve this?  Is copying the current state of these files from main and moving them over to this branch enough?

I'm also wondering if these changes are brought about from running bundle install with a different node/npm version  -- I think we had a discussion on slack about that at one point but not sure if it applies here?

Reply:
> Is copying the current state of these files from main and moving them over to this branch enough?

Yes, you could do that. You should be able to tell which commands are changing those files by running the commands and seeing what changes.